src/Bundle/AutoMapperBundle/Tests/Resources/app/config.yml

```diff
@@ -6,6 +6,8 @@ framework:
 jane_auto_mapper:
     normalizer: true
     name_converter: DummyApp\IdNameConverter
+    generate_on_warmup:
```

Comment:
I would have also liked to have an easy way to configure a full namespace or directory, so that we can say all classes in this namespace/directory will generate a mapper from/to some other class,

This would be very useful for normalization and denormalizartion in apis for instance: I know all my `App\Api\Model` will need an automapper

WDYT? I don't find any nice configuration syntax for this, any idea?